README.md

```diff
@@ -36,6 +36,45 @@ Requirements:
 <br/>
 <br/>
 
+## React Native
+<img src="react-native-screenshot.png" width="200"  alt="react_native_screenhot"/>
+
+<br/>
+
+Requirements:
+- Node 16+
+
+### Suggestions
+When using a React Native WebView the success of app2app depends on the user's default browser. The WebView by default will open app links correctly but any link that opens a new tab will be sent to the user's browser. To prevent this you can disable multiple windows.
+
+``` html
+<WebView
+      source={{ uri: url }}
+      onNavigationStateChange={this.handleWebViewNavigationStateChange}
+      setSupportMultipleWindows={false}
+    />
+```
+
+After disabling multiple windows you'll need to handle all app links manually. This can be done by adding a `onNavigationStateChange` listener and opening any non-hosted-payments-page url with the `Linking` class.
+``` javascript
+// attempt to redirect to an installed app
```

Comment:
Looks like the indentation is off in a couple places in here. Worth getting right if this is public docs.

Reply:
Ah yeah good catch